lib/puppet-syntax/tasks/puppet-syntax.rb

```diff
@@ -49,8 +49,11 @@ def initialize(*args)
 
           c = PuppetSyntax::Templates.new
           errors = c.check(filelist_templates)
-          $stdout.puts "#{errors.join("\n")}\n" unless errors.empty?
-          exit 1 unless errors.empty?
+          unless errors.empty?
```

Comment:
You probably also want output of any warnings

Reply:
Good point, thanks!
done.